Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -611,6 +611,7 @@
 	<definition name="auth/workbench/addpublicpage" extends="base.definition.workbench.nav">
 		<put-attribute name="title"
 			value="Quadriga Workbench - Add Public Page Settings" />
+		<put-attribute name="currentPage" value="settings" />	
```

Comment:
you have to keep the currentPage from before, otherwise the "Projects" tab won't be highlighted anymore.